packages/perseus/src/widgets/interaction/__snapshots__/interaction.test.ts.snap

```diff
@@ -647,7 +647,7 @@ exports[`interaction widget renders movable point elements with blank constraint
                 style="position: absolute; left: 0px; top: 0px;"
               >
                 <div
-                  style="position: absolute; width: 26px; height: 26px; left: 23.36363636363636px; top: 350.6363636363636px; transform: translateX(0px) translateY(0px) translateZ(0);"
+                  style="position: absolute; width: 26px; height: 26px; left: 23.36363636363636px; top: 350.6363636363636px; transform: translateX(0px) translateY(-72.72727272727275px) translateZ(0);"
```

Comment:
I think this change has to do with the fact that UserInputManager is causing an additional rerender. I made a story and compared it between `main` and this PR - no changes. I think this is a good example of why we don't want to test implementation details; I believe we're experiencing the same user-facing behavior but we're asserting against code.

Reply:
These snapshot tests (especially on legacy/unmaintained widgets) were mostly a very course way of ensuring we didn't break things. We also may have been _missing_ a render when we captured this snapshot. 🤔   I don't think there's much value in spending time on these widget's tests unless we can verify they're broken.